### PR TITLE
Add AMPKit JS Wrapper

### PR DESCRIPTION
**Pull request by stephen-deg**
Add a wrapper for AMPKit around the CDN creation and update the iOS project to include the script.

Next PR will actually include the AMPKit changes to convert the URLs with this script.

### CODE REVIEW

File: mobile-web/src/amp-url-creator.js [hunk 1]
```diff
@@ -64,23 +83,46 @@ const MAX_DOMAIN_LABEL_LENGTH_ = 63;
...
+ * 
+ * @param {string} url The complete publisher url.
+ * @param {object} initParams Params containing origin, etc.
+ * @param (boolean} native Whether or not the url generated follows rules for native viewers (like AMPKit)
+ * @param {string} opt_cacheUrlAuthority
+ * @param {string} opt_viewerJsVersion
+ * @return {!Promise<string>}
+ * @private
+ */
+function constructViewerCacheUrlOptions(url, native, initParams,
```
Comment (chenshay) [inline mobile-web/src/amp-url-creator.js]: change to isNative to show it's a boolean

File: mobile-web/src/amp-url-creator.js [hunk 2]
```diff
@@ -64,23 +83,46 @@ const MAX_DOMAIN_LABEL_LENGTH_ = 63;
...
+ * Constructs a Viewer cache url using these rules:
+ * https://developers.google.com/amp/cache/overview
+ * 
+ * Example:
+ * Input url 'http://ampproject.org' can return 
+ * 'https://www-ampproject-org.cdn.ampproject.org/v/s/www.ampproject.org/?amp_js_v=0.1#origin=http%3A%2F%2Flocalhost%3A8000'
+ * 
+ * @param {string} url The complete publisher url.
+ * @param {object} initParams Params containing origin, etc.
+ * @param (boolean} native Whether or not the url generated follows rules for native viewers (like AMPKit)
```
Comment (chenshay) [inline mobile-web/src/amp-url-creator.js]: should be {boolean} rather than (boolean}

Comment (stephen-deg) [reply mobile-web/src/amp-url-creator.js]: Done.

File: mobile-web/src/amp-url-creator.js (hunk 1 above)
Comment (stephen-deg) [reply mobile-web/src/amp-url-creator.js]: Done.

**Pull request merged**